File: Codes/code.cpp
// Solved by: SHOUMYA

#include <bits/stdc++.h>
using namespace std;

#define fast_io ios::sync_with_stdio(false); cin.tie(nullptr);
#define ll long long
#define endl '\n'
#define pb push_back

ll sum(ll n){
    ll s = 0;
    while(n){
        s += n%10;
        n /= 10;
    }
    return s;
}

void solve() {
    ll n,cnt = 0; cin>>n;
    for(ll i=n+1; i<=n+200;i++){
        if(i - sum(i) == n) cnt++;
    }
    cout<<cnt<<endl;
}

int main() {
    fast_io;
    #ifndef ONLINE_JUDGE
        freopen("input.txt", "r", stdin);
        freopen("output.txt", "w", stdout);
    #endif

    int t;
    // t = 1;
    cin >> t;
    for (int tc = 1; tc <= t; tc++) {
        // cout << "Case #" << tc << ": ";
        solve();
    }
    return 0;
}

File: Codes/code.py
# Solved by: SHOUMYA

import sys


LOCAL = False


def solve(tc: int) -> None:
    print("Hare Krishna")
    print(tc)


def main() -> None:
    if LOCAL:
        sys.stdin = open("input.txt", "r")
        sys.stdout = open("output.txt", "w")

    input = sys.stdin.readline

    t = int(input().strip())
    for tc in range(1, t + 1):
        # print(f"Case #{tc}:", end=" ")
        solve(tc)


if __name__ == "__main__":
    main()